bootstrap-salt.sh

```diff
@@ -3744,7 +3744,13 @@ install_centos_git_deps() {
 
     if [ "${_PY_EXE}" != "" ]; then
         # If "-x" is defined, install dependencies with pip based on the Python version given.
-        _PIP_PACKAGES="jinja2 msgpack-python pycrypto PyYAML tornado zmq"
+        _PIP_PACKAGES="m2crypto jinja2 msgpack-python pycrypto PyYAML tornado zmq"
+
+        # install swig and openssl on cent6
+        if [ "$DISTRO_MAJOR_VERSION" -eq 6 ]; then
+            __M2C_DEPS="openssl-devel swig"
+            __yum_install_noinput ${__M2C_DEPS} || return 1
```

Comment:
I think you don't need intermediate `__M2C_DEPS` variable, just append the packages strait after the yum function call.
It would also satisfy shellcheck.

Reply:
Thanks once again for the great suggestion :)